src/services/settings.ts

```diff
@@ -46,6 +47,7 @@ export const DEFAULT_USER_SETTINGS = {
     [AUTOCOMPLETE_ON_ENTER]: true,
     [IS_HOTKEYS_HELP_HIDDEN_KEY]: false,
     [AUTO_REFRESH_INTERVAL]: 0,
+    [CASE_SENSITIVE_JSON_SEARCH]: false,
```

Comment:
looks like global option but  as far as I see JSONTree used in many places

<img width="431" alt="image" src="https://github.com/user-attachments/assets/15149443-d751-46c3-aa66-2ba5db0a9764">

Reply:
I suppose case-sensitiveness should be scoped to JSONTreeWrapper (or JSONTreeSearch or whatever component it is used in)

Reply:
It's a setting that should be saved to LS. Thats why it is among other settings. Or I didn't get you?..